File: lecture_4/SumOfPrime.cpp
#include<bits/stdc++.h>
using namespace std;

bool isPrime(int n){
    if(n <2){
        return false;
    }else{
        for(int i = 2; i < n; i++){
        if(n % i == 0){
        return  false;
    }
    }
    }
    return true;
}
int sum(int n){
    int sum = 0;
    for(int i = 2; i < n; i ++){
        if(isPrime(i)){
            sum += i;
        }
    }
    return sum;
}
int main(){
    int n ;
    cin >> n;
    cout <<sum(n);
    return 0;
}

File: lecture_4/SumOfPrime.py

def isPrime(n):
    if(n < 2):
        return False
    else:
        for i in range(2, int((n) ** 1/2)+1):
            if(n % i == 0):
                return False
    return True

def sum(n):
    sum =0;
    for i in range(n):
        if(isPrime(i)):
            sum += i
    return sum

n = int(input())
print(sum(n))
